### PR TITLE
docs: small fixes

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: MAINTAIN.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The forecasting problem might be solved with an explicit priority system (like
 Bram's todo.txt). Meanwhile the Neovim priority system is defined by:
 
 * PRs nearing completion.
-* Issue labels. E.g. the `+plan` label increases the ticket's priority merely
+* Issue labels. E.g. the `has-plan` label increases the ticket's priority merely
```
Comment (zeertzjq) [inline MAINTAIN.md:28]:
```suggestion
* Issue labels. E.g. the `has:plan` label increases the ticket's priority merely
```

**Comment by clason** [comment]
Probably about ready to get merged?

**Review by clason — APPROVED** [review]

File: src/nvim/viml/parser/expressions.c [hunk 2]
```diff
@@ -2866,7 +2866,7 @@ viml_pexpr_parse_no_paren_closing_error: {}
         case kENodeOperator:
           if (prev_token.type == kExprLexSpacing) {
             // For some reason "function (args)" is a function call, but
-            // "(funcref) (args)" is not. AFAIR this somehow involves
+            // "(funcref) (args)" is not. As far as I remember this somehow involves
```
Comment (justinmk) [inline src/nvim/viml/parser/expressions.c:2869]: I do not think we should start expanding a bunch of acronyms. Use google, people. Or chatgpt.

Comment (dundargoc) [reply src/nvim/viml/parser/expressions.c:2869]: Noted.

**Pull request merged**